src/js/components/modals/ServiceFormModal.js

```diff
@@ -342,37 +344,26 @@ class ServiceFormModal extends mixin(StoreMixin) {
           return error;
         });
 
-        return (
-          <li key={path}>
-            {`${fieldId}: ${errors}`}
-          </li>
-        );
+        // Return path-prefixed error string
+        return `${fieldId}: ${errors}`;
+
       });
     }
 
     if (this.shouldForceUpdate(errorMessage)) {
       return (
-        <div className="error-field text-danger">
-          <h4 className="text-align-center text-danger flush-top">
-            App is currently locked by one or more deployments. Press the button
-            again to forcefully change and deploy the new configuration.
-          </h4>
-        </div>
+        <CollapsibleErrorMessage
+          message={'App is currently locked by one or more deployments. Press the button again ' +
```

Comment:
It was mainly for readability purposes, but I guess even if the string spans in more than 1 line HTML don't care. 😄